components/AnchorNavigation/src/index.scss

```diff
@@ -0,0 +1,46 @@
+.denhaag-anchor-navigation {
+  border-inline-start: var(--denhaag-anchor-navigation-border-inline-start);
+}
+
+.denhaag-anchor-navigation ul {
+  list-style: var(--denhaag-anchor-navigation-ul-list-style);
+  padding-inline-start: var(--denhaag-anchor-navigation-ul-padding-inline-start);
+}
+
+.denhaag-anchor-navigation ul li a {
+  font-family: var(--denhaag-anchor-navigation-anchor-font-family);
+  font-size: var(--denhaag-anchor-navigation-anchor-font-size);
+  line-height: var(--denhaag-anchor-navigation-anchor-line-height);
+  text-decoration: var(--denhaag-anchor-navigation-anchor-text-decoration);
+}
+
+.denhaag-anchor-navigation ul li a,
+.denhaag-anchor-navigation ul li a:visited {
```

Comment:
No modifier classes for storybook? My suggestion: `.denhaag-anchor-navigation__link--visited`

Reply:
This one is unnecessary in my opinion, because it's a browser thing when link is clicked, no need for using js to add this class.
Why add an exra css class if a link is used/clicked/visited?

Reply:
> This one is unnecessary in my opinion, because it's a browser thing when link is clicked, no need for using js to add this class. Why add an exra css class if a link is used/clicked/visited and handled by the browser style?

Even though you're writing tests at the moment, it's easier to do visual regression tests when there is a class name for the visited effect. We can write a test using `--visited` to ensure the pseudo selectors don't visually change.

Reply:
@bddjong @paulAcato  also when the styling is the same for the default anchor link and a:visited?
Otherwise, I wil add additional BEM html with no difference with the default.